machine_learning_hep/utilities.py

```diff
@@ -116,34 +116,26 @@ def create_folder_struc(maindir, listpath):
         if not os.path.exists(folder):
             os.makedirs(folder)
 
-def checkdirlist(dirlist):
-    """
-    Checks if list of folder already exist, to not overwrite by accident
-    """
-    exfolders = 0
-    for _, mydir in enumerate(dirlist):
-        if os.path.exists(mydir):
-            print("rm -rf ", mydir)
-            exfolders = exfolders - 1
-    return exfolders
-
-def checkdir(mydir):
```

Comment:
Do I get it right that you just changed the order of the functions? Sorry, it is not easy to see from the diff whether there are other changes.

Reply:
This might have been an artifact of the rebasing, I will have to check this